### DISCUSSION TITLE
Symbols for Csla?

### DISCUSSION THREAD

**Discussion by ajohnstone-ks** [MarimerLLC/csla#3939] [Questions] [answered]
Is there someplace to get the symbols for Csla?  I'm trying to figure out a problem with our Csla 7 update; our login command works fine, but then I execute a command (which is a continuation of the login process) but I get an error `Csla.DataPortalException : Unable to read beyond the end of the stream.
  Exception doesn't have a stacktrace`

This seems to be after the call returns from the server, because in our DataPortalController the response is 200 and it seems to exit out of there fine w/o invoking the ExceptionInspector.  But I'm a bit stuck on where the issue could be.

**Comment by hurcane** [comment]
I believe the symbols are available in the standard way using the NuGet symbol server. #1188 is the reference I am using.

Make sure your Visual Studio configuration is set to use the NuGet symbol server. Unless this has been broken recently...

**Reply by ajohnstone-ks** [reply]
It doesn't seem to find them; do I need to do something besides disable Just my code & check the Nuget.org symbol server in the list of symbol locations?

**Reply by rockfordlhotka** [reply, marked as answer]
I've never actually gotten this to work.

The nuspec definitions for CSLA include the symbols, and the symbols are pushed to NuGet, so _in theory_ it should work, but I'd sure love to have someone figure out how to actually get at them!!

**Reply by ajohnstone-ks** [reply]
@rockfordlhotka It could have been Resharper but for some reason today I did get partially decompiled symbols.  Anyway glad to know its just not me doing something wrong, thanks!
